presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -166,19 +166,70 @@ queries.
     * Checks the nullity of control sum and test sum.
     * If either control mean or test mean very close 0, checks if both are close to 0.
     * Checks the relative error between control sum and test sum.
+* **VARCHAR Columns**
+    * For ``VARCHAR`` columns, a simple checksum column is generated for verification using :func:`!checksum`.
+    * If ``validate-string-as-double`` is set, 6 additional columns below are generated. If ``NULL`` counts
+      are equal before and after casting all values to ``DOUBLE``, apply the aforementioned floating point validation.
+      Otherwise, check if the simple checksum matches.
+        * Count of the ``NULL`` values
+        * Count of the ``NULL`` values after all values are casted to ``DOUBLE``
+        * After casting all values to ``DOUBLE``
```

Comment:
```suggestion
        After casting all values to ``DOUBLE``
```
Suggest not formatting this line as an item in the list, to help the reader by showing them six items instead of the formatting suggesting there are seven.